pkg/services/ngalert/prom/convert.go

```diff
@@ -232,7 +235,8 @@ func (p *Converter) convertRule(orgID int64, namespaceUID string, promGroup Prom
 		title = rule.Alert
 	}
 
-	labels := make(map[string]string, len(rule.Labels)+len(promGroup.Labels)+1)
+	labels := make(map[string]string, len(rule.Labels)+len(promGroup.Labels)+len(p.cfg.ExtraLabels)+1)
+	maps.Copy(labels, p.cfg.ExtraLabels)
```

Comment:
Below we save these labels to `rule.Labels`. I am not sure we should include them there because, afaiu, they will be returned by`GET` endpoint 


Also, I think we should still validate it for allowed labels. For example, `__grafana_alert_rule_uid__` should not be allowed, I think

Reply:
> Below we save these labels to rule.Labels. I am not sure we should include them there because, afaiu, they will be returned byGET endpoint

That's intentional, we just add labels to the rules and return them later.

> Also, I think we should still validate it for allowed labels. For example, __grafana_alert_rule_uid__ should not be allowed, I think

We do have some standard labels validation (reserved labels) later, but having additional validation for `__grafana_alert_rule_uid__` would be useful, I'll add something.